de.metas.acct.base/src/main/sql/postgresql/ddl/functions/ProfitAndLossBalanceForAccountInPeriod.sql

```diff
@@ -0,0 +1,59 @@
+DROP FUNCTION IF EXISTS ProfitAndLossBalanceForAccountInPeriod(IN p_C_ElementValue_ID numeric, IN p_startDate timestamp, IN p_endDate timestamp);
+
+CREATE OR REPLACE FUNCTION ProfitAndLossBalanceForAccountInPeriod(IN p_C_ElementValue_ID numeric, IN p_startDate timestamp, IN p_endDate timestamp)
+    RETURNS NUMERIC
+AS
+$BODY$
+
+WITH periods_between_dates AS
+         (
+             SELECT --
+                    p.c_period_id,
+                    p.startdate,
+                    p.enddate
+             FROM c_period p
+             WHERE TRUE
+               AND p.startdate >= p_startDate
+               AND p.enddate <= p_endDate
+         ),
+     fact_acct_summary_for_periods AS
+         (
+             -- fact_acct_summary filtered for selected period
+             SELECT fas.*
+             FROM periods_between_dates p
+                      INNER JOIN fact_acct_summary fas ON fas.c_period_id = p.c_period_id
```

Comment:
do we have an index on fas.c_period_id?`if not, do we need one?
EDI: we have one

Reply:
Since we already have an index, there is nothing i have to do, right?

Reply:
Assuming there's nothing else for me to do and marking as resolved